sdks/java/core/src/main/java/org/apache/beam/sdk/schemas/Schema.java

```diff
@@ -238,17 +238,35 @@ public FieldType type() {
    */
   @AutoValue
   public abstract static class FieldType implements Serializable {
-    // Returns the type of this field.
```

Comment:
All these comment changes seem unrelated to a Json Deserialzier. Separate commit?